zebra/zapi_msg.c

```diff
@@ -2511,6 +2511,19 @@ static inline void zread_ipset_entry(ZAPI_HANDLER_ARGS)
 		if (zpi.proto != 0)
 			zpi.filter_bm |= PBR_FILTER_PROTO;
 
+		if (!(zpi.dst.family == AF_INET
+		      || zpi.dst.family == AF_INET6)) {
+			zlog_warn("Unsupported PBR IP family: %s (%" PRIu8 ")",
```

Comment:
agree with @srimohans that it would be handy to distinguish in log message if that for src or dst. I mean src/dst keyword would be useful to have in log messages.

Reply:
I think this should be part of an other PR. this is out of scope of the fix, I tend to say, if you dont mind